lib/elven_gard/helpers/basic_encoder.ex

```diff
@@ -1,4 +1,4 @@
-defmodule ElvenGard.Helpers.PacketEncoder do
+defmodule ElvenGard.Helpers.BasicEncoder do
```

Comment:
A better way to structure your code should be `ElvenGard.Encoder` unless its an intermediate level Encoder implementation such as 
```
:ranch_protocol -> :cowboy_clear -> :cowboy_http
```

Reply:
I'll change it, thanks